queries.sql

```diff
@@ -47,3 +47,11 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, count(neutered) as result FROM animals GROUP BY neutered ORDER BY result DESC LIMIT 1;
 SELECT species, MAX(weight_kg) AS max_weight, MIN(weight_kg) AS min_weight FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT animals.name, owners.full_name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE full_name='Melody';
+SELECT animals.name AS animal_name, species.name AS species_type FROM animals JOIN species ON animals.species_id=species.id WHERE species.name='Pokemon';
+SELECT full_name, animals.name FROM owners LEFT JOIN animals ON animals.owner_id=owners.id;
+SELECT count(animals) FROM animals JOIN species ON animals.species_id=species.id;
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE owners.full_name='Jennifer Orwell';
+SELECT species.name, count(animals) FROM animals JOIN species ON animals.species_id=species.id GROUP BY species.name;
+SELECT owners.full_name , COUNT(animals.name) AS total_animals FROM owners JOIN animals ON animals.owner_id=owners.id GROUP BY owners.full_name ORDER BY total_animals DESC LIMIT 1;
```

Comment:
- [ ] You should list all animals owned by Dean Winchester that haven't tried to escape.

From the project requirements :arrow_down: 
> - List all animals owned by Dean Winchester that haven't tried to escape.